tgui/packages/tgui/interfaces/PersonalSummary.tsx

```diff
@@ -0,0 +1,217 @@
+  const {
+    is_antagonist,
+    is_part_time,
+    is_escaped,
+    is_pilot,
+    base_wage,
+    score_adjusted_wage,
+    objective_completed_bonus,
+    all_objectives_bonus,
+    pilot_bonus,
+    earned_spacebux,
+    total_spacebux,
+    held_item,
+  } = spacebux_data;
+  return (
+    <Section title="Spacebux">
+      <Table lineHeight="1.2em">
+        {!is_antagonist && (
+          <>
+            <Table.Row className="candystripe">
+              <Table.Cell>
+                Base Wage {!!is_part_time && '(part-time)'}
+              </Table.Cell>
+              <Table.Cell fontWeight="bold" textAlign="right">
+                {base_wage}
+              </Table.Cell>
+            </Table.Row>
+            <Table.Row className="candystripe">
+              <Table.Cell>Station Grade Tax</Table.Cell>
+              <Table.Cell fontWeight="bold" textAlign="right">
+                -{base_wage - score_adjusted_wage}
+              </Table.Cell>
+            </Table.Row>
+            {!is_escaped && (
+              <Table.Row className="candystripe">
+                <Table.Cell>Did Not Escape</Table.Cell>
+                <Table.Cell fontWeight="bold" textAlign="right">
+                  -{score_adjusted_wage - earned_spacebux}
+                </Table.Cell>
+              </Table.Row>
+            )}
+            {objective_completed_bonus > 0 && (
+              <Table.Row className="candystripe">
+                <Table.Cell>Crew objective bonus</Table.Cell>
+                <Table.Cell fontWeight="bold" textAlign="right">
+                  +{objective_completed_bonus}
+                </Table.Cell>
+              </Table.Row>
+            )}
+            {all_objectives_bonus > 0 && (
+              <Table.Row className="candystripe">
+                <Table.Cell>All crew objective bonus</Table.Cell>
+                <Table.Cell fontWeight="bold" textAlign="right">
+                  +{all_objectives_bonus}
+                </Table.Cell>
+              </Table.Row>
+            )}
+          </>
+        )}
+
+        {!!is_antagonist && (
+          <>
+            <Table.Row className="candystripe">
+              <Table.Cell>Base Wage</Table.Cell>
+              <Table.Cell fontWeight="bold" textAlign="right">
+                {earned_spacebux}
+              </Table.Cell>
+            </Table.Row>
+            <Table.Row className="candystripe">
+              <Table.Cell colSpan={2}>Antagonist - No tax!</Table.Cell>
+            </Table.Row>
+          </>
+        )}
+
+        {is_pilot && (
+          <Table.Row className="candystripe">
+            <Table.Cell>Pilot&apos;s bonus</Table.Cell>
+            <Table.Cell fontWeight="bold" textAlign="right">
+              +{pilot_bonus}
+            </Table.Cell>
+          </Table.Row>
+        )}
+      </Table>
+      <hr />
+      <Table>
+        <Table.Row>
+          <Table.Cell fontSize="1.1em" fontWeight="bold">
+            PAYOUT
```

Comment:
I'm not sure about using caps as text emphasis. Also I'd check out LabeledList, they might work with candystripe too